.github/workflows/schedule.yml

```diff
@@ -25,7 +25,7 @@ jobs:
             skip-checks: true
         run: |
           # Hard-code user config
-          git config user.email "team-hammerhead@snyk.io"
+          git config user.email "team-cli@snyk.io"
```

Comment:
Question: Does this exist?

Reply:
Updated in https://github.com/snyk/actions/pull/136/commits/c68f83879ba2df9a66d3fd602da61d9402c49d75